test/libmpv_encode.c

```diff
@@ -47,13 +47,13 @@
 // Global handle
 static mpv_handle *ctx;
 // Temporary output file
-static const char *out_path;
+static char *out_path;
 
 static void exit_cleanup(void)
 {
     if (ctx)
         mpv_destroy(ctx);
-    if (out_path && *out_path)
+    if (out_path)
```

Comment:
Why remove this check?

Reply:
I swear this used to segfault on that pointer but now it apparently doesn't. hmm.